index.js

```diff
@@ -1,34 +1,40 @@
-'use strict';
+'use strict'
+
+const defaultPackages = [
+  'apollo-cache',
+  'apollo-cache-inmemory',
+  'apollo-client',
+  'apollo-link',
+  'apollo-link-context',
+  'apollo-link-http',
+  'graphql',
+  'graphql-tools',
+  'graphql-tag',
+];
 
 module.exports = {
   name: 'ember-apollo-client',
 
-  included() {
+  included(app) {
     this._super.included.apply(this, arguments);
-
+    
+    let config = app.project.config(app.env) || {};
+    this.addonConfig = config['apollo'] || {};
+    
     this.import('vendor/-apollo-client-bundle.js');
     this.import('vendor/-apollo-client-shims.js');
   },
-
+  
   treeForVendor() {
     const WebpackDependencyPlugin = require('./lib/webpack-dependency-plugin');
-
+    const userSpecifiedPackages = this.addonConfig.include;
+    
     return new WebpackDependencyPlugin({
       outputName: 'apollo-client',
-      expose: [
-        'apollo-cache',
-        'apollo-cache-inmemory',
-        'apollo-client',
-        'apollo-link',
-        'apollo-link-context',
-        'apollo-link-http',
-        'graphql',
-        'graphql-tools',
-        'graphql-tag'
-      ]
-    });
+      expose: [...defaultPackages, ...userSpecifiedPackages],
```

Comment:
What do you think about having a default list of packages, but then if the user specifies any packages we *only* use the ones they've specified? The goal would be to enable users to remove as well as add packages.